types/cesium/index.d.ts

```diff
@@ -2528,7 +2530,7 @@ declare namespace Cesium {
           availability?: TimeIntervalCollection;
           show?: boolean;
           description?: Property;
-          position?: PositionProperty;
+          position?: CallbackProperty | PositionProperty;
```

Comment:
The documentation does not specify that `CallbackProperty` works for this: https://cesium.com/docs/cesiumjs-ref-doc/Entity.html?classFilter=entity

Reply:
Well it works. and every property can be CallbackProperty by definition of the CallbackProperty